executor/analyze.go

```diff
@@ -1001,6 +1001,8 @@ func (e *AnalyzeFastExec) buildHist(ID int64, collector *statistics.SampleCollec
 	// build CMSketch
 	var ndv, scaleRatio uint64
 	collector.CMSketch, ndv, scaleRatio = statistics.NewCMSketchWithTopN(defaultCMSketchDepth, defaultCMSketchWidth, data, 20, uint64(rowCount))
+	// Scale the total column size.
+	collector.TotalSize *= rowCount / int64(len(collector.Samples))
```

Comment:
This `TotalSize` computed in `CalcTotalSize` contains the flag byte, which is inconsistent with normal analyze as:
```golang
func (c *SampleCollector) collect(sc *stmtctx.StatementContext, d types.Datum) error {
    if !c.IsMerger {
        if d.IsNull() {
            c.NullCount++
            return nil
        }
        // Minus one is to remove the flag byte.
        c.TotalSize += int64(len(d.GetBytes()) - 1)
    }
```

Reply:
Values here are already decoded, there are no flags in it.